.github/workflows/scala.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       with:
         java-version: 1.8
     - name: Run tests
-      run: sbt test
+      run: sbt clean; sbt testAll
```

Comment:
I struggled to define a dependency between IntegratonTest/test task and assembly that would force running assembly and refreshing the artifacts. Therefore, it's possible that a stale jar will be used for testing unless we run "clean" before running the tests. 

If there's a way to force the dependency so that the IntegrationTest/test task always reassembles the extension jar, I'd be happy to incorporate it.